.circleci/config.yml

```diff
@@ -913,6 +752,9 @@ workflows:
     when:
       equal: [ scheduled_pipeline, << pipeline.trigger_source >> ]
     jobs:
+      - 'lint-linux'
+      - 'lint-macos'
+      - 'lint-windows'
       - 'test-go-linux'
       - 'test-go-linux-386'
       - 'test-go-mac'
```

Comment:
Maybe add a `requires` so the tests are only run when the linters are ok?

Reply:
While that could prevent us from needing to run on macOS and windows, this would extend the total time by at least 12mins or so. For now these can run in parallel while we gather additional information.